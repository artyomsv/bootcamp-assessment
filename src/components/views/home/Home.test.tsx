import * as React from 'react';
import {createShallow} from '@material-ui/core/test-utils';
import HomeView from './Home.view';

const shallow = createShallow();

describe('renders without crashing', () => {
  test('if renders', () => {
    const mockFn = jest.fn();
    const result = shallow(
      <HomeView
        navigateToMovies={mockFn}
        navigateToActors={mockFn}
      />
    );

    expect(result).toMatchSnapshot();
  });
});
