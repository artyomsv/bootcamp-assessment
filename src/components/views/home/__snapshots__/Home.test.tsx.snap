// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing if renders 1`] = `
<HomeView
  classes={
    Object {
      "headlines": "HomeView-headlines-5",
      "link": "HomeView-link-4",
      "linkText": "HomeView-linkText-6",
      "paper": "HomeView-paper-3",
      "root": "HomeView-root-1",
      "view": "HomeView-view-2",
    }
  }
  navigateToActors={[MockFunction]}
  navigateToMovies={[MockFunction]}
/>
`;
